src/products/components/ProductList/messages.ts

```diff
@@ -14,3 +14,22 @@ export const messages = defineMessages({
     description: "product publication date"
   }
 });
+
+export const columnsMessages = defineMessages({
+  availability: {
+    defaultMessage: "Availability",
+    description: "product channels"
+  },
+  price: {
+    defaultMessage: "Price",
+    description: "product price"
+  },
+  type: {
+    defaultMessage: "Type",
+    description: "product type"
+  },
+  updatedAt: {
+    defaultMessage: "Updated At",
```

Comment:
Wouldn't "Last updated" be better? "Updated at 2 months ago" sounds wrong. CC @pwgryglak

Reply:
To add to that, we should also refrain from capitalising words in sentence Like This as it makes them less readable.